src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,106 @@
+package storage;
+
+import parsers.DateTimeParser;
+import tasks.Task;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Todo;
+import ui.Ui;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
```

Comment:
maybe can add javadocs for classes and not just methods to show the general function of the class